content/source/docs/cloud/workspaces/run-tasks.html.md

```diff
@@ -5,93 +5,91 @@ page_title: "Run Tasks - Workspaces - Terraform Cloud and Terraform Enterprise"
+Run tasks execute between the [plan](../run/states.html#2-the-plan-stage) and [apply](../run/states.html#6-the-apply-stage) stage of a [Terraform run](../run/index.html). Once all run tasks executions have completed, the run will end based on the most restrictive enforcement level that has been configured in each associated run tasks.
 
-The Pre-Apply stage will always end with the most restrictive status of the tasks configured to run. For example, if a mandatory task fails and an advisory task succeeds, the stage will fail and the run will error. If an advisory task fails but a mandatory task succeeds, the stage will succeed and the run will proceed to the apply stage.
+For example, if a mandatory task fails and an advisory task succeeds, the run will fail. If an advisory task fails, but a mandatory task succeeds, the run will succeed and proceed to the apply stage. Regardless of the exit status of a task, the status and any related message data will be displayed in the UI.
 
-Regardless of the exit status of a task, the status and any related message will be displayed in the UI.
-
-Here is an example of a run that failed due to a mandatory task.
+Here is an example of a run that failed due to a mandatory run task.
 
 ![Screenshot: a run with failed tasks](./images/run-tasks-run-failed.png)
 
 And here is an example of a run that succeeded.
 
-![Screenshot: a run with failed tasks](./images/run-tasks-run-success.png)
+![Screenshot: a run with passed tasks](./images/run-tasks-run-success.png)
+
+## Removing a Run Task from a Workspace
+
+1. Navigate to the desired workspace, then click **Run Tasks** from the **Settings** menu.
 
-## Deleting a Task from a Workspace
+2. Click the ellipses (...) on the associated run task, and then click **Remove**.
 
-1. Navigate to the desired workspace, then select "Tasks" from the "Settings" menu.
+    ![Screenshot: removing a run task from a workspace](./images/run-tasks-workspace-remove.png)
 
-2. Click Remove next to the task you wish to delete
 
+## Deleting a Run Task
 
-## Deleting a Task Event Hook
+1. Navigate to **Settings** and **Run Tasks**
 
-1. Navigate to Settings and Task event hooks
+2. On the run task you want to delete, click the ellipses and then click **Edit**
 
-2. Click on Edit for the event hook you wish to modify
+3. Click **Delete run task**
 
-3. Click on Delete event hook
+   ![Screenshot: a warning when attempting to delete an in-use run task](./images/run-tasks-delete-run-task-warning.png)
 
-You can not delete task event hooks that are still attached to workspaces. If you attempt this, you will see a warning in the UI containing a list of any workspaces that are consuming the event hook. You must remove the task from the impacted workspaces before proceeding with the deletion.
+You cannot delete run tasks that are still associated with a workspace. If you attempt this, you will see a warning in the UI containing a list of all workspaces that are associated with the run task. 
```

Comment:
I would move this up to come right after **Delete** and before the image showing the warning.